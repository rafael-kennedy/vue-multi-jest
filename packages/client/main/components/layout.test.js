const subject = require(`./layout.vue`).default
const { mount } = require(`vue-test-utils`)

describe(`scaffolds out page`, () => {
  const wrapper = mount(subject)

  test(`has all components`, () => {
    ;[`main`, `aside`, `header`, `footer`].forEach(tag => {
      expect(wrapper.contains(tag)).toBeTruthy()
    })
  })
  test(`snapshot`, () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})
