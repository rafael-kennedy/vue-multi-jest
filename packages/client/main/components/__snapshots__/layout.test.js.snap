// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[
  `scaffolds out page snapshot 1`
] = `"<div><header></header> <aside></aside> <main></main> <footer></footer></div>"`
